docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -93,17 +96,27 @@ As the component is fully state-dependent, you can customize the data it shows,
 fun MessageListHeader(
     channel: Channel,
     currentUser: User?,
-    connectionState = ConnectionState.CONNECTED,
-    messageMode: MessageMode = Normal,
-    ..., // content
-    ... // actions
+    modifier: Modifier = Modifier,
+    typingUsers: List<User> = emptyList(),
+    messageMode: MessageMode = MessageMode.Normal,
+    connectionState: ConnectionState = ConnectionState.CONNECTED,
+    color: Color = ChatTheme.colors.barsBackground,
+    shape: Shape = ChatTheme.shapes.header,
+    elevation: Dp = ChatTheme.dimens.headerElevation,
+    ..., // Handlers
+    ... // Content
 )
 ```
 
 * `channel`: The information about the current channel, used to show the member count, name and avatar data.
```

Comment:
I think this should be `watchersCount`

Reply:
Seems like it can be either the number of members or the number of watchers: https://github.com/GetStream/stream-chat-android/blob/3c293ecd081790180e3f89b6d7bfd608f5b6eb5a/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt#L72

Reply:
Alright, then ignore, it's good as-is